plantio/urls.py

```diff
@@ -1,18 +1,23 @@
 from django.conf.urls import url
 from django.urls import include
 
-from plantio.views.plantio import PlantioAPIView
-from plantio.views.aplicacao import AplicacaoAgrotoxicoAPIView
+from plantio.views import plantio, aplicacao
 
 
 urlpatterns = [
     url(r'^plantio/', include([
-        url(r'^$', PlantioAPIView.as_view(), name='plantio-create'),
+        url(r'^$', plantio.PlantioAPIView.as_view(), name='plantio-create'),
 
         url(r'^aplicar/', include([
             url(
-                r'agrotoxico/$', AplicacaoAgrotoxicoAPIView.as_view(),
+                r'agrotoxico/$', aplicacao.AplicacaoAgrotoxicoAPIView.as_view(),
                 name='plantio-associar'),
         ])),
+
+        url(r'^(?P<idPropriedade>[0-9A-Za-z]{8}-[0-9A-Za-z]{4}-4[0-9A-Za-z]{3}-[89ABab][0-9A-Za-z]{3}-[0-9A-Za-z]{12})/historico/', include([
```

Comment:
Acho que seria bom abstrair esse regex pra ser reutilizado em outras rotas

Reply:
esse regex tá na norma do uuid? acho que o default não funciona nele 00000000-0000-0000-0000-000000000000 que seria valido. Além disso, as letras vão de a-f e não de a-z.

https://stackoverflow.com/a/6640851

Reply:
Agradeço os comentários Thiago, quanto a rota, também preferi colocar um plantio/historico/<idPropriedade> porque poderíamos utilizá-la para quando for idTalhao "plantio/historico/<idPropriedade", a única infelicidade no primeiro comentário é que essa task resolve a US24 que trata do histórico de plantações da propriedade e não do talhão que é tratado na US25, nossas US estão mal escritas, então as que estão separadas vamos respeitar né kkkk, então nesse quesito ela está com a responsabilidade correta de retornar todo o histórico de uma propriedade.

Reply:
O regex foi retirado de um padrão que eu havia achado na internet, não lembro-me de ter averiguado em algum outro local, mas vou trabalhar nisso e subir as correções, tmj!

Reply:
Ah sim, entendi. Já que é os plantios da propriedade, então a rota ficaria melhor para os padroes rest assim:
`propriedade/{idPropriedade}/plantios`

Reply:
Acho que juntar o historido do talhao e da propriedade vai gerar uma complexidade mais alta doq o necessario, é bem melhor separar em duas com responsabilidade unicas:
`talhao/{idTalhao}/plantios` ou simplesmente `talhao/{idTalhao}/`